### DISCUSSION TITLE
Tabs moved to incorrect panel after sidebar closed and reopened?

### DISCUSSION THREAD

**Discussion by kellyclowers** [mbnuqw/sidebery#705] [General]
Possibly related to https://github.com/mbnuqw/sidebery/issues/703 ? Though that sounds like it affects things at time of tab creation...

I had a number of tabs open in different panels, however after accidentally closing the sidebar (cmd-b is unfortunately close to cmd-v) and reopening it, I found all tabs not in the main/first panel had been moved to the second panel, which also happens to be the only panel with a regex move pattern (though I am just using simple URL match, no actual regex special chars)

I was pretty upset, but it turned out it at least preserved the tabs in panel order, so it was pretty simple to mass select them and move them to correct panel... would have been ugly if they were all out of order. 

Still, this surely shouldn't happen at all?

How they should be (after I moved them back):
![Screen Shot 2022-07-25 at 13 30 58](https://user-images.githubusercontent.com/82614936/180872720-55ed8fc5-5535-4c39-b657-e310ce91b7a4.png)

How it was when I first re-opened the sidebery sidebar (actually the panels were hidden, I opened new tabs in them before I figured out what was going on and thought to take a screenshot):
![Screen Shot 2022-07-25 at 13 20 37](https://user-images.githubusercontent.com/82614936/180872830-51a92e6d-8e9a-4d3b-85a8-50c16873cb75.png)

**Comment by albino1** [comment]
Could be related to the various other panel fail bug reports.  Some commits were made recently to try and fix those:

https://github.com/mbnuqw/sidebery/commit/720ff2f22f81892edf875ac2ec39851fd754774c
https://github.com/mbnuqw/sidebery/commit/afecf35abd57408b85e5738e71858298ae791058

@mbnuqw any idea when a new 4.10.2 release will come out with those fixes included to see if it solves any of these types of problems?

**Comment by mbnuqw** [comment]
@albino1, I hope it'll be ready by the end of this week (bugs with snapshots, #703, testing) + a few days for reviewing in AMO

**Reply by albino1** [reply]
Awesome, thank you, looking forward to it!
